XLMOD.obo

```diff
@@ -10312,3 +10312,16 @@ synonym: "TNBSA" EXACT []
 synonym: "Picrylsulfonic acid" EXACT []
 is_a: XLMOD:06010 ! LCMS derivatization reagent
 relationship: is_reactive_with XLMOD:00028 ! primary amine reactive
+
+[Term]
+id: XLMOD:09317
+name: DSAU
+def: "Urea crosslinker – C2-arm, NHS ester (DSAU)" [PSI:XL]
+synonym: "C2-Urea-NHS-crosslinker" EXACT []
+synonym: "Urea-crosslinker C2-succinimide" EXACT []
+synonym: "DSAU" EXACT []
+property_value: deadEndFormula: "C 13 H 14 N 4 O 9" xsd:string
+property_value: monoIsotopicMass: "140.02219198" xsd:double
+is_a: XLMOD:00005 ! homofunctional cross-linker
+relationship: has_reactive_group XLMOD:00101 ! NHS ester
+relationship: is_cleavable XLMOD:00067 ! CID cleavable C-S bond
```

Comment:
XLMOD:00067 is also the wrong term here - need to check for the right one

Reply:
the only terms that look like they would be accurate are `XLMOD:00018 ! cleavable by MS2 labile bond` as the most general term or `XLMOD:00035 ! CID cleavable` (assuming it cleaves in in CID as well). 

I don't think we have a term for CO-NH bond cleavable /peptide like cleavage (I assume that is where it cleaves) - we could add a term for that as well (copied from the C-S bond term)

```
[Term]
id: XLMOD:000174
name: CID cleavable CO-NH-S bond
def: "Indicates that a cross-linker contains a CID (Collision Induced Dissociation) cleavable peptide bond." [PSI:XL]
is_a: XLMOD:00035 ! CID cleavable
relationship: is_cleavable XLMOD:00018 ! cleavable by MS2 labile bond
```

But somebody with a better authority on chemical/MS nomenclature would need to decide what name/description to use